src/gui/properties/peerlistwidget.cpp

```diff
@@ -86,6 +86,7 @@ uint qHash(const PeerEndpoint &peerEndpoint, const uint seed = 0)
 PeerListWidget::PeerListWidget(PropertiesWidget *parent)
     : QTreeView(parent)
     , m_properties(parent)
+    , m_showConnectingPeers(u"PeerList/ShowConnectingPeers"_qs, false)
```

Comment:
Currently all peers are showed so it should be `true` by default.

Reply:
the default is changed to not show connecting peers anymore.

Reply:
Changing the default value will be unexpected for users, won't it?
Just let user to choose it by himself.

Reply:
Current behaviour is incorrect. Reported peers count differ from peers in peers list for no apparent reason

Reply:
> Current behaviour is incorrect.

I wouldn't say it is incorrect.
The peer list simply displays extended information including connecting peers, the number of which does not matter to be displayed in the transfer list.

Okay, I won't mind if others approve of changing the default value.

Reply:
Current behaviour is confusing*.

Reply:
If we showed all peers and added a column to indicate connection status, that would solve all confusions. I don’t think many people would understand or check the flags.